libs/extensions/angular/image-banner/src/image-banner.component.scss

```diff
@@ -5,6 +5,25 @@ $blur-radius: 60px;
 $max-content-width: 324px;
 $main-image-height-small: 132px;
 $main-image-height-large: 164px;
+$cq-width: 515px;
+
+@mixin cq-min() {
```

Comment:
consider a more descriptive name, it loses a bit of meaning like this.
Maybe `container-below-min-width` and `container-above-min-width`